Automattic/vip-go-mu-plugins
# Add VIP Search widget

## Description

This PR introduces a WordPress widget that can be used as a drop-in UI element for VIP Enterprise Search. It is based in Jetpack's Search Widget, but this first iteration doesn't include ES facets (tags).

## Changelog Description

### Add VIP Search Widget

Adding a first iteration of a Search Widget for VIP Search.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Make sure VIP Search is enabled.
3. Set up a Search Widget in wp-admin.
4. Perform a search in the FE.
